### ISSUE TITLE
[BUG][IPC4] failed to bind modules mixin.5.1 -> mixout.2.1 when playback on HDA Analog and DeepBuffer simultaneously

### ISSUE THREAD

**Issue by miRoox** [thesofproject/sof#6265]
**Describe the bug**
CI caught this issue when testing multiple pause-resume. The scenario can be simplify to playback on HDA Analog and DeepBuffer simultaneously.

CI report ID: 15173

**To Reproduce**
For TGLU_UP_HDA, run
```
aplay -D hw:0,0 -r 48000 -c 2 -f S16_LE -vv -i /dev/zero -q
```
and
```
aplay -D hw:0,1 -r 48000 -c 2 -f S16_LE -vv -i /dev/zero -q
```

**Reproduction Rate**
100%

**Environment**
1) Branch name and commit hash of the 2 repositories: sof (firmware/topology) and linux (kernel driver).
    * Kernel: https://github.com/thesofproject/linux/commit/91a177700173
    * FIrmware: IPC4 tester
2) Name of the topology file
    * Topology:
cavs-mixin-mixout-hda.tplg
![cavs-mixin-mixout-hda.tplg](https://user-images.githubusercontent.com/20090991/188770421-34e2c7ef-4589-4384-8953-266f339d45e9.png)
cavs-sdw.tplg 
![cavs-sdw.tplg](https://user-images.githubusercontent.com/20090991/188770493-c1439e01-c3ce-4ce7-9348-827078457a35.png)
3) Name of the platform(s) on which the bug is observed.
    * Platform: TGLU_RVP_HDA/ADLP_RVP_SDW

**Screenshots or console output**

**dmesg**
```
[ 2352.791858] kernel: snd_sof:sof_ipc4_route_setup: sof-audio-pci-intel-tgl 0000:00:1f.3: bind mixin.5.1 -> mixout.2.1
[ 2352.791860] kernel: snd_sof:sof_ipc4_log_header: sof-audio-pci-intel-tgl 0000:00:1f.3: ipc tx      : 0x13010003|0x0: GLB_SET_PIPELINE_STATE
[ 2352.792081] kernel: snd_sof:sof_ipc4_log_header: sof-audio-pci-intel-tgl 0000:00:1f.3: ipc tx reply: 0x33000000|0x0: GLB_SET_PIPELINE_STATE
[ 2352.792086] kernel: snd_sof:sof_ipc4_log_header: sof-audio-pci-intel-tgl 0000:00:1f.3: ipc tx done : 0x13010003|0x0: GLB_SET_PIPELINE_STATE
[ 2352.792093] kernel: snd_sof:sof_ipc4_log_header: sof-audio-pci-intel-tgl 0000:00:1f.3: ipc tx      : 0x45010002|0x3: MOD_BIND
[ 2352.792094] kernel: snd_sof_intel_hda_common:hda_dsp_stream_trigger: sof-audio-pci-intel-tgl 0000:00:1f.3: FW Poll Status: reg[0x160]=0x140000 successful
[ 2352.792277] kernel: snd_sof:sof_ipc4_log_header: sof-audio-pci-intel-tgl 0000:00:1f.3: ipc tx reply: 0x65000006|0x3: MOD_BIND
[ 2352.792279] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: FW reported error: 6 - Unknown error while processing the request
[ 2352.792284] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: ipc error for msg 0x45010002|0x3
[ 2352.792285] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: sof_ipc4_route_setup: failed to bind modules mixin.5.1 -> mixout.2.1
```

[full_dmesg.txt](https://github.com/thesofproject/linux/files/9503175/full_dmesg.txt)

**Comment by RanderWang** [comment]
how about close source fw ?

**Comment by lgirdwood** [comment]
@serhiy-katsyuba-intel fyi.

**Comment by miRoox** [comment]
It seems this issue also got fixed by #6264. We'll monitor it for a while.

**Comment by miRoox** [comment]
Cannot reproduce it.
